### ISSUE TITLE
Exporting TERM in /etc/profile breaks screen/tmux

### ISSUE THREAD

**Issue by ghost** [msys2/MSYS2-packages#486]
the TERM environmental variable is set to xterm-256color in MSYS2-packages/filesystem/profile will override changes made by applications. One example of this is screen and tmux which need to set TERM to screen/screen-256color to work properly. I assume this is done to make mintty look prettier so I'd suggest configuring it within a mintty configuration file so avoid overriding TERM in all circumstances.

**Comment by elieux** [comment]
Agreed. I was trying to get some reports on our ML whether there's any downside to this and I haven't gotten around to looking at the topic again since. Also see https://sourceforge.net/p/msys2/tickets/241/.

**Comment by lazka** [comment]
This was fixed long ago in 8db3812e0877e6920bfcf9191020ebf2bab4e650
